playbooks/base-minimal-test/pre.yaml

```diff
@@ -22,6 +22,8 @@
       include_role:
         name: start-zuul-console
 
+- hosts: controller
```

Comment:
Please read the comments in zuul.d/jobs.yaml, it explains how to properly test trusted jobs.

Reply:
Yes but appliance nodes don't get the authorized_keys updated since `add-build-sshkey` ignore the appliance nodes.

```
- hosts: all:!appliance
  tasks:
    - name: Run add-build-sshkey role
      include_role:
        name: add-build-sshkey
```

So basically I can put the vcenter:

- in `appliance`, and I lose the ssh access
- in a new group like `vcenters`, but git is not available and I don't want the workspace on the node anyway
- in `controller: same situation.

Reply:
How we did this for network appliances, is used default user / password for 'zuul' user, then generated ssh keys, then delete password. I suggest we do that for now, and look to update add-build-sshkey role, as I don't expect it to work properly for vmware.  Basically, appliances need to use nested ansible in pre-run phase to setup this.

Reply:
Currently, I cannot do pw auth over SSH with the vcenter other the ESXi, so basically I lose the connectivity. `add-build-sshkey` should work fine with ESXi as soon as we can specify an alternative location for the `authorized_keys` file.

Reply:
okay, we'll have to enable that in image builds, that is what we do in network. That is going to be the lesser of 2 evils here, because to modify our trusted jobs and update add-build-sshkey for an 'appliance' is going to work without some major refactoring.

Reply:
Well technically speaking, I don't really need to do any ssh connection. We can do everything over SSH. It was more to help for the troubleshooting. I will inject my SSH key in the images. This way, I will be able to connect on the hosts directly.